Ginger/GingerCoreCommon/Actions/ActOutDataSourceConfig.cs

```diff
@@ -18,28 +18,29 @@
 
 using Amdocs.Ginger.Repository;
 using Amdocs.Ginger.Common;
+using System.Collections.Generic;
 
 namespace GingerCore.Actions
 {
     public class ActOutDataSourceConfig : RepositoryItemBase
     {
         private bool mActive;
         public enum eOutputType
-        {    
-            Parameter,            
-            Path,            
+        {
+            Parameter,
+            Path,
             Actual,
             Parameter_Path
         }
 
-        public  static partial class Fields
+        public static partial class Fields
         {
             public static string DSName = "DSName";
             public static string DSTable = "DSTable";
-            public static string Active = "Active";
-            public static string OutputType = "OutputType";
-            public static string TableColumn = "TableColumn";
-            public static string PossibleValues = "PossibleValues";
+            //public static string Active = "Active";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=4815237)